dashboard/test/controllers/api/v1/pd/workshops_controller_test.rb

```diff
@@ -405,6 +405,13 @@ class Api::V1::Pd::WorkshopsControllerTest < ::ActionController::TestCase
     assert_response :forbidden
   end
 
+  test 'created_at is included in a serialized workshop response' do
```

Comment:
This attribute is getting special treatment :) If we have to do this more often, we can consolidate all serialization tests in a WorkshopSerializerTest class.